operator/v1/types_network.go

```diff
@@ -100,12 +100,79 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// IPAMConfig configures IPAM module will be used for IP Address Management(IPAM).
+	IPAMConfig IPAMConfig `json:"ipamConfig"`
```

Comment:
needs to be pointer, and `omitempty`. (also, space before the "`(`" in the comment)

Reply:
changed.